README.md

```diff
@@ -1,3 +1,29 @@
-# NostrSDK
+# Nostr SDK iOS
 
-A description of this package.
+[Nostr](https://github.com/nostr-protocol/nostr) SDK library for iOS applications.
+
+## Features
+
+TBD
+
+## Spec Compliance
+
+Nostr SDK iOS implements the following NIPs:
+
+TBD
+
+## Installation
+
+TBD
+
+## Contributing
+
+If you would like to contribute to this library, please see [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Contact
+
+These are the core maintainers of this library and their Nostr public keys.
+
+- [Bryan Montz](https://github.com/bryanmontz) (npub1qlk0nqupxmlyxravg0aqscxmcc4q4tq898z6x003rykwwh3npj0syvyayc)
```

Comment:
I tried linking the npubs with `nostr:` but GitHub's Markdown renderer doesn't seem to like it, so I left them as plaintext.